horovod/common/stall_inspector.h

```diff
@@ -29,7 +29,11 @@ class ResponseCache;
 
 class StallInspector {
 public:
-  StallInspector() = delete;
+  /* because of default constructor of 'StallInspector'
```

Comment:
I do not understand this change. In your commit message you quote an error message applying to 
```
StallInspector() = default;
```
But the class doesn't contain that line on Horovod master! It's been 
```
StallInspector() = delete;
```
for a while now and that doesn't have this problem.